packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -127,6 +137,36 @@ describe('logs', () => {
 
       expect(server.requests.length).toEqual(1)
     })
+
+    it('should send bridge event when bridge is present', () => {
+      updateExperimentalFeatures(['event-bridge'])
+      const sendSpy = spyOn(initEventBridgeStub(), 'send')
+
+      const sendLog = startLogs()
+      sendLog(DEFAULT_MESSAGE, {})
+
+      expect(server.requests.length).toEqual(0)
+      expect(sendSpy).toHaveBeenCalled()
+    })
+  })
+
+  describe('sampling', () => {
+    it('should be applied when event bridge is present', () => {
```

Comment:
I thought we agree to not apply the sampling on logs for now?

Reply:
we agreed to recheck it with product and finally went with sampling at each init like we do in other env without session/cookie.
Any concern?

Reply:
Alright, I find this a bit weird product wise but other than that it's fine